app/src/ui/commit-message/commit-message-dialog.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react'
+import { Dispatcher } from '../dispatcher'
+import { Repository } from '../../models/repository'
+import { Dialog, DialogContent } from '../dialog'
+import { ICommitContext } from '../../models/commit'
+import { CommitIdentity } from '../../models/commit-identity'
+import { ICommitMessage } from '../../models/commit-message'
+import { IAutocompletionProvider } from '../autocompletion'
+import { IAuthor } from '../../models/author'
+import { CommitMessage } from '../changes/commit-message'
+interface ICommitMessageDialogProps {
```

Comment:
Nit: should we leave a blank line above this one?
```suggestion

interface ICommitMessageDialogProps {
```

Reply:
I always thing there is a space when I see this. :P 
![image](https://user-images.githubusercontent.com/75402236/117798935-9ea0ca00-b21f-11eb-9b21-8c2683853215.png)